app/src/installer.js

```diff
@@ -26,20 +26,25 @@ global.installer = function() {
     install: function(zxpPath) {
 
       console.log("using target path of " + target_path());
-      console.log("startig to install ZXP from path " + zxpPath);
+      console.log("starting to install ZXP from path " + zxpPath);
 
       return promise = new Promise(function(resolve, reject) {
         var spawn = install_process.spawn(path.join(__dirname, target_path()), [CMD_PREFIX+"install", zxpPath]);
 
-        // AEM only prints out if something went wrong. Adobe - go figure
```

Comment:
I took this out because it's not specifically true. It's also fairly common for CLIs to only print results when errors happen.